core/common/src/test/scala/net/liftweb/common/BoxSpec.scala

```diff
@@ -152,14 +152,17 @@ class BoxSpec extends Specification with ScalaCheck with BoxGenerator {
         Full(Empty).flatten must_== Empty
       }
     }
+    "define a 'mapFailure' method returning itself." in {
+      Full(1).mapFailure(identity) must_==  Full(1)
```

Comment:
Yeah, I did that get the niggling feeling when I wrote this. You are right. That could be confusing. I'll change it to something more appropriate.